marquez_airflow/models.py

```diff
@@ -30,8 +30,26 @@ def __repr__(self):
                           {self.description!r},{self.ordinal_position!r})"
 
 
+class DbTableName:
+    def __init__(self, value: str):
+        parts = value.split('.')
+        if len(parts) > 2:
+            raise ValueError(
+                f"Table {value} must contain the schema and table name "
+                " separated by '.', or table name only."
+            )
+        self.value = parts[1] if len(parts) == 2 else parts[0]
```

Comment:
I was expecting this object to contain the fully qualified name.
IE : the schema and the name

Reply:
Yeah, I went back and forth on what exactly the class should encapsulate.I think I'd make sense to have `schema `, `name`, `qualified_name` (which would include the schema and table name)